junit-5/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'java'
+}
+
+repositories { mavenCentral() }
+
+dependencies {
+    implementation 'org.junit.jupiter:junit-jupiter-api:5.9.1'
+    implementation 'org.junit.jupiter:junit-jupiter-engine:5.9.1'
+    implementation 'org.junit.platform:junit-platform-console:1.8.1'
+    implementation 'org.seleniumhq.selenium:selenium-java:4.22.0'
+    implementation 'com.browserstack:browserstack-local-java:1.0.6'
+    implementation 'com.googlecode.json-simple:json-simple:1.1.1'
+    implementation 'org.yaml:snakeyaml:2.2'
+    implementation 'com.browserstack:browserstack-java-sdk:latest.release'
```

Comment:
With `compileOnly` - `jvmArgs "-javaagent:${browserstackSDKArtifact.file}"` is not working - it needs a complete jar path